README.md

```diff
@@ -1,26 +1,15 @@
-# Deprecation notice
-
-This module was designed for Puppet versions 2 and 3. It should work also on Puppet 4 but doesn't use any of its features.
-
-The current Puppet 3 compatible codebase is no longer actively maintained by example42.
-
-Still, Pull Requests that fix bugs or introduce backwards compatible features will be accepted.
-
-
 # Puppet module: zabbix_agent
 
-This is a Puppet module for zabbix_agent based on the second generation layout ("NextGen") of Example42 Puppet Modules.
+This is a Puppet module for zabbix_agent based on the third generation layout ("NewGen") of Example42 Puppet Modules.
```

Comment:
NewGen?? Let's avoid another unfortunate naming, maybe just mention if for Puppet 4 onwards.